tests/teamcity/run-server.sh

```diff
@@ -46,6 +46,11 @@ curl -s $FXA_PROFILE_VERSION
 curl -s $FXA_AUTH_VERSION
 echo ""
 
+echo ""
+echo "Selenium version:"
+curl -s http://127.0.0.1:4444/wd/hub/status | python -mjson.tool
```

Comment:
Not quite following. Piping to `cat` would have the same output. 

But I was going for the pretty-printing, and `json.tool` is there if python is there.